### PR TITLE
[WIP Draft]Agent installation

### PR DESCRIPTION
**Pull request by Amrita42**
**Scope/Objective**   
Create a table of content for the **Agent-based Installation** book which is part of the **Installing** book. You can see the placeholder for Agent-based Installation content in the preview below. Currently, this is a work in progress and I do have a number of action items as well as queries. 
Please take a look at the proposed TOC and provide your feedback.

Initially, I am looking at the following as a starting point:

- [Agent-9](https://issues.redhat.com/browse/AGENT-9)
- [Agent-5](https://issues.redhat.com/browse/AGENT-5)
- [Agent-6](https://issues.redhat.com/browse/AGENT-6)

[Preview](http://file.del.redhat.com/asakthiv/agent-install/installing/installing-with-agent/installing-with-agent.html)

### CODE REVIEW

**Comment by Amrita42** [comment]
curl -H 'Content-Type: application/json' --request POST --data '{"PR_BRANCH":"Amrita42:agent-install","BASE_REPO":"openshift/openshift-docs","PR_NUMBER":"47887","USER_NAME":"Amrita42","BASE_REF":"main","REPO_NAME":"Amrita42/openshift-docs"}' https://eoa6vg2jiwjbnh6.m.pipedream.net

File: installing/installing-with-agent/installing-with-agent.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+toc::[]
+
+== Agent installation
+Agent installation is based on the technology of Assisted Installer (AI), that is link:https://console.redhat.com[console.redhat.com] and the Infrastructure Operator. It is a command line tool that generates an image that runs the Assisted Service and the Assisted Installer Agent.
+//TODO>>include telemetry GA
+
+It is a subcommand that resides in the {product-title} installer and also utilizes the {product-title} installer for the input asset management.
+
+This subcommand generates a live {op-system-first} ISO with an embedded Ignition config containing `systemd` units to start the Assisted service, Agent, network, etc. that are running in containers pulled from the {product-title} release payload.
```
Comment (rwsu) [inline installing/installing-with-agent/installing-with-agent.adoc:15]: It also contains a service that starts the cluster installation if all the required hosts have registered with assisted-service and passed validation.

File: installing/installing-with-agent/installing-with-agent.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+The Agent installation utilizes Zero Touch Provisioning (ZTP) custom resources. ZTP allows you to provision new edge sites with declarative configurations of bare-metal equipment at remote sites.
```
Comment (rwsu) [inline installing/installing-with-agent/installing-with-agent.adoc:16]: We have also added a mode where the installer takes in higher level manifests, install-config.yaml and agent-config.yaml, to generate the ZTP resources. I think we prefer to have users start with the higher level manifests.

Comment (Amrita42) [reply installing/installing-with-agent/installing-with-agent.adoc:16]: ok noted.  https://issues.redhat.com/browse/AGENT-248

File: installing/installing-with-agent/installing-with-agent.adoc (hunk 1 above)
Comment (Amrita42) [reply installing/installing-with-agent/installing-with-agent.adoc:15]: Could you please share the related jira?

**Comment by Amrita42** [comment]
Closing this PR . Please refer to https://github.com/openshift/openshift-docs/pull/48673  for further discussion.  @rwsu

**Pull request closed without merging**